### PR TITLE
﻿CU-86a21gwp3 - NEON3 - RECEIVE Tokens - Your Address Screen

### PR DESCRIPTION
**Pull request by Thacryba**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-86a21gwp3 NEON3 - RECEIVE Tokens - Your Address Screen](https://app.clickup.com/t/86a21gwp3)

File: src/renderer/src/routes/modals/SelectAccount/index.tsx [hunk 1]
```diff
@@ -10,15 +9,18 @@ import { useBalancesAndExchange } from '@renderer/hooks/useBalancesAndExchange'
 import { useModalNavigate, useModalState } from '@renderer/hooks/useModalRouter'
 import { useWalletsSelector } from '@renderer/hooks/useWalletSelector'
 import { EndModalLayout } from '@renderer/layouts/EndModal'
+import { KeyPrefix } from 'i18next'
 
 type TLocationState = {
   onSelectAccount: (contact: IAccountState) => void
+  keyPrefix: KeyPrefix<'modals'>
```
Comment (raulduartep) [inline src/renderer/src/routes/modals/SelectAccount/index.tsx]: Pass a title and buttonLabel instead, this makes it more reusable, as you won't need to create something within the modal lang for it to work.

File: src/renderer/src/routes/pages/Send/SelectAccount.tsx [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
 import { Separator } from '@renderer/components/Separator'
 import { StyleHelper } from '@renderer/helpers/StyleHelper'
 import { useModalNavigate } from '@renderer/hooks/useModalRouter'
+import { KeyPrefix } from 'i18next'
 
 type TAccountParams = {
   selectedAccount?: IAccountState
   onSelectAccount: (contact: IAccountState) => void
   active: boolean
+  keyPrefix: KeyPrefix<'pages'>
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Send/SelectAccount.tsx]: Same comments of SelectAccountModal

File: src/renderer/src/routes/pages/Receive/SelectYourReceivingAddress.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+              <QRCodeSVG id="QRCode" size={172} value={address} includeMargin />
+            ) : (
+              <TbQrcode className="text-green-700 w-[140px] h-[140px]" />
+            )}
+          </div>
+        </div>
+      </div>
+      <Button
+        className="mt-10 w-[16rem]"
+        type="submit"
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Receive/SelectYourReceivingAddress.tsx]: Remove this type, it is not necessary

**Pull request merged**